.vib/rails/goss/rails.yaml

```diff
@@ -0,0 +1,12 @@
+command:
```

Comment:
I would add a check for the database-related gems added by us at compilation time.